applications/structure/beam/application.h

```diff
@@ -205,10 +207,7 @@ class Application : public ApplicationBase<dim, Number>
         if(dim == 3)
           repetitions[2] = this->repetitions2;
 
-        dealii::GridGenerator::subdivided_hyper_rectangle(*this->grid->triangulation,
```

Comment:
Looks like the use of `this->grid->triangulation` inside `lambda_create_triangulation()` for this particular application is a bug in the current implementation!?